app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,53 @@
+class CommentsController < ApplicationController
+  # index
+  def index
+    @comment = Comment.all
+  end
```

Comment:
do we need this action? At any point will you want to see all of the comments in your entire application? I think we'll only ever want to see comments associated with a post